selectors.yml

```
@@ -24,9 +42,15 @@ selectors:
   338:
     selector: 2995292832068775165
     name: "cronos-testnet"
+  398:
+    selector: 5061593697262339000
+    name: near-testnet
   420:
     selector: 2664363617261496610
     name: "ethereum-testnet-goerli-optimism-1"
+  462:
+    selector: 7317911323415911000
+    name: areon-network-testnet
```

Maybe for consistency with `gnosis_chain` and friends?
Or even drop the `_network` entirely, as most projects here do have network in their names (unless there's precedent deployments using that name)
```suggestion
    name: areon_network-testnet
```